Alloy/lib/alloy.js

```diff
@@ -417,7 +417,18 @@ exports.createWidget = function(id, name, args) {
  * @return {Alloy.Controller} Alloy controller object.
  */
 exports.createController = function(name, args) {
-	return new (require('alloy/controllers/' + name))(args);
+	try {
+		return new (require('alloy/controllers/' + name.replace(/^\//, '')))(args);
```

Comment:
I like the idea replacing those additional slashes at the beginning of the controller file name - but do we have any examples about the failures while loading controllers ?